src/components/CameraCapture.svelte

```diff
@@ -0,0 +1,92 @@
+<script lang="ts">
+    import {ImageCapture} from 'image-capture'
+    import {onDestroy, onMount} from "svelte";
+    import Button from "./Button.svelte";
+    import jsQR, {QRCode} from "jsqr";
+    import type {NanoAddress} from "../machinery/models";
+    import {tools} from "nanocurrency-web";
+    import {softwareKeysStore} from "../stores/stores";
+
+    export let scannedAddress: (address: NanoAddress) => void
+
+    let videoPreview: MediaStream | undefined
+    let showVideo: boolean = true
+
```

Comment:
Code below cannot be classified as very clean, but it gets the job done. We can probably clean this up later though.